docs/gds-supported-platform/making-things-private-in-harbor.md

```diff
@@ -0,0 +1,65 @@
+Making images private in Harbor
+===============================
+
+DCS has a need for private Docker images. By default, the GSP Harbor
```

Comment:
Maybe don't need to call out DCS in the generic docs

Reply:
```suggestion
Sometimes, we need to make our Docker images private. By default, the GSP Harbor
```